modules/ada/2.9.2/MODULE.bazel

```diff
@@ -0,0 +1,5 @@
+module(
+    name = "ada",
+    version = "2.9.2",
+    compatibility_level = 1,
```

Comment:
please add `bazel_compatibility = [">=7.2.1"],` here, for the usage of overlay files